kexgssc.c

```diff
@@ -453,11 +458,16 @@ kexgssgex_client(struct ssh *ssh)
 			do {
 				type = ssh_packet_read(ssh);
 				if (type == SSH2_MSG_KEXGSS_HOSTKEY) {
+					char *tmp = NULL;
```

Comment:
```suggestion
					u_char *tmp = NULL;
```

Otherwise this results in a C type error.